indico/web/client/js/jquery/widgets/jinja/sui_remote_search_dropdown_widget.js

```diff
@@ -0,0 +1,26 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {WTFRemoteSearchDropdown} from 'indico/react/components';
+
+window.setupRemoteSearchDropdownWidget = function setupRemoteSearchDropdownWidget({
+  fieldId,
+  ...props
+}) {
+  // Make sure the dropdown is displayed above the dialog
+  const field = $(`#${fieldId}`);
+  field.closest('.ui-dialog-content').css('overflow', 'inherit');
+  field.closest('.exclusivePopup').css('overflow', 'inherit');
+
+  ReactDOM.render(
+    <WTFRemoteSearchDropdown fieldId={fieldId} {...props} />,
```

Comment:
since you need all the props including field it, matbe just accept `props` in the function in line 13 and use `props.fieldId` in line 18